sensor/common/pubsub/lane/default.go

```diff
@@ -0,0 +1,145 @@
+package lane
+
+import (
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/sensor/common/pubsub"
+	"github.com/stackrox/rox/sensor/common/pubsub/consumer"
+	pubsubErrors "github.com/stackrox/rox/sensor/common/pubsub/errors"
+)
+
+type DefaultConfig struct {
+	Config
+}
+
+func WithDefaultLaneSize(size int) pubsub.LaneOption {
+	return func(lane pubsub.Lane) {
+		laneImpl, ok := lane.(*defaultLane)
+		if !ok {
+			return
+		}
+		laneImpl.size = size
+	}
+}
```

Comment:
Missing validation for negative values. Providing `-1` would cause a panic. Need to think how to handle 0.